lib/Controllers/create.js

```diff
@@ -30,6 +57,9 @@ Create.prototype.write = function(req, res, context) {
         res.header('Location', location);
       }
 
+      // Reload for that all field are present
+      return instance.reload();
```

Comment:
isn't this going to have the same problem as this: https://github.com/dchester/epilogue/commit/6671404bf4c49befea5da59cb4f839901b52f810 ?